packages/abi-typegen/src/abi/types/EvmAddressType.ts

```diff
@@ -0,0 +1,28 @@
+import type { IType } from '../../types/interfaces/IType';
+
+import { AType } from './AType';
+
+export class EvmAddressType extends AType implements IType {
+  public static swayType = 'struct EvmAddress';
+
+  public name = 'evmAddress';
+
+  static MATCH_REGEX: RegExp = /^struct EvmAddress$/m;
+
+  static isSuitableFor(params: { type: string }) {
+    return EvmAddressType.MATCH_REGEX.test(params.type);
+  }
+
+  public parseComponentsAttributes(_params: { types: IType[] }) {
+    const capitalizedName = 'EvmAddress';
+
+    this.attributes = {
+      inputLabel: capitalizedName,
+      outputLabel: capitalizedName,
+    };
+
+    this.requiredFuelsMembersImports = ['Address', capitalizedName];
```

Comment:
Is `Address` really needed here?

Unlike `EvmAddress`, it doesn't seem to be required in the [generated DTS](https://github.com/FuelLabs/fuels-ts/pull/995/files#diff-19032ebe72a0da3b4fed02b9a04ed3803e0a189b450a60f1bc9c5330e0635234) file.

Reply:
On second thoughts it definitely isn't required, introduced it as an import so that `Address` _could_ be used for `EvmAddress` conversions. Will remove 👍🏻